src/autocomplete/databases/clickhouse/tests/grant/grant-jdbc.test.ts

```diff
@@ -0,0 +1,21 @@
+import {parseClickHouseQueryWithCursor} from '../../index';
+
+test('should not report errors', () => {
+    const autocompleteResult = parseClickHouseQueryWithCursor(
+        `
+          GRANT ON CLUSTER test_cluster
+            JDBC
+          ON *.* TO test_user1, test_user2
+          WITH GRANT OPTION
+          WITH REPLACE OPTION;|
+        `,
+    );
```

Comment:
We should not use `WithCursor` if it's in the end and you just check that there are no errors, checkout how Robert does it https://github.com/gravity-ui/websql-autocomplete/blob/main/src/autocomplete/databases/redis/tests/append/append.test.ts#L4